docs/best_practices/integration_checklist.rst

```diff
@@ -60,7 +63,8 @@ Adding Your Tool to the IUC Repository
 - Within that folder, create a corresponding branch. You might name it after the tool.
 - After you have tested your tool and are completely happy with it (per
   previous sections of this document), add your tool and all associated data,
-  then Commit the changes with ``git commit``
+  then Commit the changes with ``git commit``. Finally push your changes
+  to github with ``git push``.
```

Comment:
``git push origin <branch name>`` is the recommend way, isn't it?
A few lines above we should write `git checkout -b <branch name>`